extensions/amp-accordion/1.0/accordion.js

```diff
@@ -72,8 +81,73 @@ export function Accordion({
     });
   }, [expandSingleSection]);
 
+  useImperativeHandle(
+    ref,
+    () => {
+      return {
+        toggle,
+        expand,
+        collapse,
+      };
+    },
+    // Does not like expandedMap, but w/o it does not work
+    [toggle, collapse, expand, expandedMap]
+  );
+
+  const toggle = useCallback(
+    (id) => {
+      if (id) {
+        if (realIdMap.current[id]) {
+          toggleExpanded(realIdMap.current[id]);
+        }
+      } else {
```

Comment:
Is this `else` really the part of our API? IMHO looks really unexpected. Same for all other `}else{`

Reply:
Correct, this is part of the `0.1` API as discussed offline.